### DISCUSSION TITLE
Git schedule

### DISCUSSION THREAD

**Discussion by sbonaime** [requarks/wiki#3688] [Error / Bug Report]
After a 2.0.0-rc.17 fresh install and import from V1, when changing the git schedule from 5 minutes to 0 minutes and 5 hours, I can see a red pop up with  "Unexpected error" for few seconds then I have a green pop up with "Storage configuration saved successfully."

<img width="848" alt="Capture d’écran 2019-10-29 à 29-10-19 15 45 38" src="https://user-images.githubusercontent.com/750709/67778298-42be8800-fa63-11e9-9fa7-a438077bc4b8.png">

**Comment by snoleo** [comment]
I encountered the same problem. 
Debian GNU/Linux 10 (buster)
(PostgreSQL) 11.5 (Debian 11.5-1+deb10u1)
node v10.16.2
git version 2.20.1

And cannot modify the default interval 5 minutes. After changing github schedule interval to eg. 15 minutes, it will pop up with "Unexpected error", then shortly pop up "Storage configuration saved successfully."
Then click save again, now the setting is 15 minutes, but after refreshing the page, the setting will be back to the default 5 minutes.

PS: Git sync works ok.

**Comment by monsterzack5** [comment]
Same exact problem here:
 - OS: requarks/wiki:2 Docker image
 - HostOS: Debian Bullseye
 - Wiki.js version: 2.0.12
 - Database engine: PostgreSQL version 11.6 using the postgres:11-alpine Docker image

**Comment by highestpeak** [comment]
Same exact problem too
